client/src/App.tsx

```diff
@@ -28,15 +28,15 @@ function App(): JSX.Element {
               <Route path="/" component={NavBar} />
               <Route exact path="/login" component={Login} />
               <Route exact path="/signup" component={Signup} />
-              <ProtectedRoute exact path="/fileUpload" component={DesignSubmit} />
+              <ProtectedRoute exact path="/fileUpload/:id" component={DesignSubmit} />
```

Comment:
Please use hyphenated case for route names with multiple words

Reply:
changed